src/build.cake

```diff
@@ -46,7 +46,7 @@ var nodeBinaryUrl = "https://github.com/sourcegraph/node-binaries/raw/main/v20.1
 var nodeArmBinaryUrl = "https://github.com/sourcegraph/node-binaries/raw/main/v20.12.2/node-win-arm64.exe";
 
 // The latest tag of the stable release from the cody repository https://github.com/sourcegraph/cody/tags
-var codyStableReleaseTag = "vscode-v1.38.1";
+var codyStableReleaseTag = "vscode-v1.38.3";

```

Comment:
```suggestion
var codyStableReleaseTag = "vscode-v1.38.3";

```

Update the Cody repo version tag or commit hash here that you want Visual Studio extension to build off of.